dask/Dockerfile

```diff
@@ -11,38 +11,27 @@ ARG UCX_PY_VER=unset
 
 COPY environment.yml /rapids.yml
 ADD https://raw.githubusercontent.com/dask/dask/main/continuous_integration/environment-$PYTHON_VER.yaml /dask.yml
-
 ADD https://github.com/rapidsai/gha-tools/releases/latest/download/tools.tar.gz /tools.tar.gz
-RUN tar -xzvf /tools.tar.gz -C /usr/local/bin --strip-components=1 \
-    && rm /tools.tar.gz
-
-RUN conda config --set ssl_verify false
-
-RUN rapids-mamba-retry install conda-merge git
-
-RUN cat /rapids.yml \
-    | sed -r "s/CUDA_VER/$(echo $CUDA_VER | cut -d. -f1,2)/g" \
-    | sed -r "s/RAPIDS_VER/${RAPIDS_VER}/g" \
-    | sed -r "s/UCX_PY_VER/${UCX_PY_VER}/g" \
-    > /rapids_pinned.yml
 
-# unpin problematic CI dependencies
-RUN cat /dask.yml \
-    | sed -r "s/pyarrow=/pyarrow>=/g" \
-    | sed -r "s/pandas=/pandas>=/g" \
-    | sed -r "s/numpy=/numpy>=/g" \
-    > /dask_unpinned.yml
-
-RUN conda-merge /rapids_pinned.yml /dask_unpinned.yml > /dask.yml
-
-RUN rapids-mamba-retry env create -n dask --file /dask.yml
-
-# Clean up pkgs to reduce image size and chmod for all users
-RUN chmod -R ugo+rw /opt/conda \
+RUN tar -xzvf /tools.tar.gz -C /usr/local/bin --strip-components=1 \
```

Comment:
Is this an overly aggressive merging of these layers? The initial problem this aimed to solve was storage pressure around making the conda install and `chmod`ing of the conda files separate layers

Reply:
Might want to consider using Dockerfile `heredocs`?

- https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/
- https://docs.docker.com/engine/reference/builder/#example-running-a-multi-line-script

Reply:
Thanks for the suggestion! Done with https://github.com/rapidsai/dask-build-environment/pull/105/commits/d1141c1d6ab4e4d1cd35a898ca1ba969ec47abc1 - https://github.com/rapidsai/dask-build-environment/pull/105/commits/4a0f0f37d504897cea119338f985ab2f4f3618e2